hera_sim/tests/test_vis.py

```diff
@@ -40,10 +97,10 @@ def test_shapes(uvdata, simulator):
     v = simulator(
         uvdata=uvdata,
         sky_freqs=np.unique(uvdata.freq_array),
-        sky_intensity=I_sky
+        sky_intensity=I_sky,
     )
 
-    assert v.simulate().shape == (NTIMES, 1, NFREQ, 1)
+    assert v.simulate().shape == (NTIMES*len(uvdata.get_antpairs()), 1, NFREQ, 1) ############################ADDED NUMBASELINES
```

Comment:
More of a comment on coding style: if you're going to place a comment immediately after a piece of code, a single hash followed by the comment should be just fine:
```
line of code # brief description of line
```
Additionally, `git` keeps track of changes between commits, so comments noting changes like this are not necessary (in fact, we would want to *remove* these comments before merging with `master`, since they'd mainly act as a source of confusion/distraction for other users looking through the source code).

Reply:
Done.